src/ray/object_manager/plasma/store.cc

```diff
@@ -382,12 +398,18 @@ void PlasmaStore::RemoveGetRequestsForClient(const std::shared_ptr<Client> &clie
   // It shouldn't be possible for a given client to be in the middle of multiple get
   // requests.
   RAY_CHECK(get_requests_to_remove.size() <= 1);
-  for (GetRequest *get_request : get_requests_to_remove) {
+  for (const auto get_request : get_requests_to_remove) {
     RemoveGetRequest(get_request);
   }
 }
 
-void PlasmaStore::ReturnFromGet(GetRequest *get_req) {
+void PlasmaStore::ReturnFromGet(std::shared_ptr<GetRequest> get_req) {
```

Comment:
maybe we should use `const std::shared_ptr<GetRequest> &get_req` here because the caller already hold the reference.

Reply:
Great catch. Fixed.

Reply:
I realized `RemoveGetRequest` was the same, so I will fix it.